l3build/l3build.lua

```diff
@@ -1383,29 +1383,31 @@ end
 --
 
 -- An auxiliary used to set up the environmental variables
-function runtool(envvar, command)
+function runtool(subdir, dir, envvar, command)
   return(
     run(
-      typesetdir,
+      typesetdir .. "/" .. subdir,
```

Comment:
As with a previous commit, don't we end up with an incorrect trailing `/` here? (I wonder about dividing up the typesetting into directories: couldn't things be done by working through the directories and copying back the results as required without needing to copy the files in the structure?)